From kernel-janitors-bounces@lists.linux-foundation.org Thu Jun 28 10:16:37 2007
From: Suresh Jayaraman <sjayaraman@novell.com>
Date: Thu, 28 Jun 2007 11:16:30 -0600
Subject: USB: ohci-pnx4008: Remove unnecessary cast of return value of kzalloc
To: <kernel-janitors@lists.osdl.org>
Message-ID: <46843A46020000F40000A9BA@lucius.provo.novell.com>
Content-Disposition: inline


Remove unnecessary cast of return value of kzalloc() in
usb/host/ohci-pnx4008.c

Signed-off-by: Suresh Jayaraman <sjayaraman@novell.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ohci-pnx4008.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/ohci-pnx4008.c
+++ b/drivers/usb/host/ohci-pnx4008.c
@@ -134,7 +134,7 @@ static int isp1301_attach(struct i2c_ada
 {
 	struct i2c_client *c;
 
-	c = (struct i2c_client *)kzalloc(sizeof(*c), GFP_KERNEL);
+	c = kzalloc(sizeof(*c), GFP_KERNEL);
 
 	if (!c)
 		return -ENOMEM;
